docs/react/faq.en-US.md

```diff
@@ -59,56 +61,56 @@ Try [Space](https://ant.design/components/space/) component to make them aligned
 
 ### antd overrides my global styles
 
-Yes, antd is designed to develop a complete background application, we override some global styles for styling convenience, and it can't be removed now. More info at https://github.com/ant-design/ant-design/issues/4331 .
+Yes, antd is designed to help you develop a complete background application. To do so, we override some global styles for styling convenience, and currently these cannot be removed or changed. More info at https://github.com/ant-design/ant-design/issues/4331 .
 
 Alternatively, follow the instructions in [How to avoid modifying global styles?](docs/react/customize-theme#How-to-avoid-modifying-global-styles-?)
 
 ### I cannot install `antd` and `antd`'s dependencies in mainland China.
 
-Long live the Girl Friend Wall! And try [cnpm](http://npm.taobao.org/).
+Long live the Great Firewall! To potentially solve this, try [cnpm](http://npm.taobao.org/).
```

Comment:
Hahahaha, there is no Great Firewall. Just keep it.

Reply:
or just remove it. keep `To potentially solve this, try [cnpm](http://npm.taobao.org/).` to sync with chinese verison doc.